src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -4,11 +4,28 @@
 
 package frc.robot.subsystems;
 
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+//import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.motorcontrol.Spark;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class Shooter extends SubsystemBase {
   /** Creates a new Shooter. */
-  public Shooter() {}
+  
+  CANSparkMax shooter;
+  
+  public Shooter() {
+    shooter = new CANSparkMax(4, MotorType.kBrushless);
```

Comment:
Angerey why are we doing magic numbers

Reply:
brandon drinking too much angry juice :(